learn/cookbooks/strapi_v4.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: Strapi v4 guide
+description: Learn how to use Meilisearch with Strapi v4.
+---
+
+# Strapi v4 guide
+
+This tutorial will show you how to integrate Meilisearch with [Strapi](https://strapi.io/) to create a search-based web app. We are going to use Strapi’s quick start guide to create a restaurant collection, and then use Meilisearch to search through our collection data.
+
+## Prerequisites
+
+Before you start, make sure you have the following installed on your machine:
+- [Node.js](https://nodejs.org/): active LTS or maintenance LTS versions, currently Node.js >=18.0.0 \<=20.x.x
+- npm >=6.0.0 (installed with Node.js) 
+- A running instance of Meilisearch (v >= 1.x). If you need help with this part, you can consult the [Installation section](/learn/getting_started/installation). 
+
+## Create a back-end using Strapi
+
+Create a directory called `my-app` where you will add our back and front-end parts of the application. Generate a back-end API using Strapi inside `my-app`:
```

Comment:
I suggest creating a couple of h3 headings in this section to make it a bit easier to navigate.